homeassistant/components/mqtt/light/schema_json.py

```diff
@@ -423,77 +429,31 @@ async def _subscribe_topics(self):
 
         last_state = await self.async_get_last_state()
         if self._optimistic and last_state:
-            self._state = last_state.state == STATE_ON
+            self._attr_is_on = last_state.state == STATE_ON
             last_attributes = last_state.attributes
```

Comment:
The following `last_attributes.get` lines might look cleaner if you hoist `last_attributes.get` into a local, á la

```
last_get = last_state.attributes.get
```

but I get it if that's against the usual code style...

Reply:
Good idea, but for for this PR we want to avoid to ucm changes.